### PR TITLE
Define the names of the label keys as constants

### PR DESCRIPTION
**Pull request by mortent**
This is useful for people who wants to use the App CRD as part of other projects/products 

@kow3ns 
@barney-s

### CODE REVIEW

**Comment by crimsonfaith91** [comment]
/lgtm
Should we add documentation to README.md about this?

File: pkg/controller/application/application_controller.go [hunk 1]
```diff
@@ -37,6 +37,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	NameLabelKey = "app.kubernetes.io/name"
+	VersionLabelKey = "app.kubernetes.io/version"
```
[the remark this answers is not in the document]
Comment (mortent) [reply pkg/controller/application/application_controller.go]: I just wasn't aware of the full list of labels. But I think we should add them all, so I have updated the PR.

**Comment by crimsonfaith91** [comment]
/lgtm
Sorry for overlooking the other variables.

**Comment by ant31** [comment]
/lgtm

As a side note: would it make sense to add those constants directly in kubebuilder?

**Pull request merged**